kubernetes_platform/python/test/snapshot/data/create_mount_delete_existing_pvc.py

```diff
@@ -24,7 +24,7 @@ def comp():
 @dsl.pipeline
 def my_pipeline():
     pvc1 = kubernetes.CreatePVC(
-        pvc_name='static-pvc-name',
+        pvc_name='static-pvc-name-cmdep',
```

Comment:
Why is this name change needed by the way?

Reply:
Please see: https://github.com/kubeflow/pipelines/pull/11831#issuecomment-2831021108